Core/Core/Planner/GetCalendarEvents.swift

```diff
@@ -20,21 +20,29 @@ import Foundation
 
 public class GetCalendarEvents: CollectionUseCase {
     public typealias Model = CalendarEvent
-    public var cacheKey: String? { "\(context.pathComponent)/calendar-events/\(type.rawValue)" }
-    public let context: Context
+    public var cacheKey: String? { "\(contexts.first?.pathComponent ?? "")/calendar-events/\(type.rawValue)" }
+    public let contexts: [Context]
     public let type: CalendarEventType
+    private let importantDates: Bool
 
-    public init(context: Context, type: CalendarEventType = .event) {
-        self.context = context
+    public init(context: Context, type: CalendarEventType = .event, importantDates: Bool = false) {
```

Comment:
We should check if this property will cause any CoreData collisions or not. Imaginary scenario:
- Go to K5 homeroom, fetch calendar events with important dates.
- Go to calendar, the same API call executes but without important dates.
- Go back to homeroom and important dates disappeared because the calendar screen's API fetch erased them from CoreData.